ui/js/models/resultset.js

```diff
@@ -191,9 +191,33 @@ treeherder.factory('ThResultSetModel', ['$rootScope', '$http', '$location',
                 return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri);
             },
 
-            triggerMissingJobs: function (resultset_id, repoName) {
+            triggerMissingJobs: function (resultset_id, repoName, decisionTaskID) {
                 var uri = resultset_id + '/trigger_missing_jobs/';
-                return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri);
+                return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri).then(function () {
```

Comment:
This just tries the old way first since it actually tries both of the methods no matter what. Now that pulse_actions is turned off, all of the old-way stuff is actually useless, so I can remove it later. I just want to leave it in for now in case we turn pulse_actions back on.

I'll make a note to myself to remove it in a few weeks.